Makefile

```
@@ -75,3 +79,10 @@ www: \
 	dist/apple-touch-icon.png \
 	dist/index.html \
 	dist/manifest.webmanifest
+
+
+.PHONY: serve
+serve: node_modules
+	$(WEBPACK_DEV_SERVER) --config $(WEBPACK_CONFIG) --open --port $(PORT)
```

```suggestion
	$(NODE ) ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --config $(WEBPACK_CONFIG) --open --port $(PORT)
```

This is probably better, so you are explicit on what interpreter actually runs it (otherwise the shebang on the CLI might spawn another Node.js) and inline `WEBPACK_DEV_SERVER`, as you are only using it here?